lib/pbkit/pbkit.c

```diff
@@ -2761,7 +2776,10 @@ void pb_kill(void)
 }
 
 
-
+void pb_set_color_format(unsigned int fmt, bool swizzled) {
+    pb_ColorFmt = fmt;
+    assert(swizzled == false);
```

Comment:
Support for swizzled buffers should be added in the future.  I prepared the API to be able to handle this.